README.md

```diff
@@ -74,19 +75,13 @@ jobs:
         uses: actions/checkout@v2
 
       - name: Use OCaml ${{ matrix.ocaml-version }}
-        uses: avsm/setup-ocaml@v1
+        uses: avsm/setup-ocaml@v2
         with:
           ocaml-version: ${{ matrix.ocaml-version }}
 
-      - run: opam pin add hello.dev . --no-action
-
-      - run: opam depext hello --yes --with-doc --with-test
-
-      - run: opam install . --deps-only --with-doc --with-test
+      - run: opam install . --deps-only
 
       - run: opam exec -- dune build
-
-      - run: opam exec -- dune runtest
```

Comment:
I'd like to see a testing workflow by default, this is the oportunity to "recommend" a workflow to ocaml newcomers:

```suggestion
      - run: opam install . --deps-only --with-test

      - run: opam exec -- dune build

      - run: opam exec -- dune runtest
```

Reply:
I was thinking about the same thing, but from v2, opam pin and depext run automatically (You can optionally disable it), so I intentionally removed it to keep the example very minimal. (If partial revert the changes, we have to add `--with-test` to the `opam-depext-flanges` input.)

Reply:
Otherwise, we can disable these functionalities by default. Because it's mainly for large projects.

Reply:
If it can be enabled I'd like to see how to use these advanced option.

Usually for testing there are no depexts needed, so having the tests would still work. Starting with opam 2.1.0 would the depexts be automatically installed with `opam install`?

Reply:
Definitely. Users should see what they are. Yes, in general it's not required, but it's often needed, so I'd like to put it to avoid confusion.

> Starting with opam 2.1.0 would the depexts be automatically installed with `opam install`?

IIRC, opam-depext as a package will be deprecated and will just be bundled with opam. CC: @rjbou

Reply:
yes, depext is integrated in opam 2.1 (not yet release, soon a release candidate), but opam 2.0 still uses `opam-depext` plugin. Ftm it is frozen, and once 2.1 released it will be slowly deprecated.

Reply:
By the way, if setup-ocaml attempts to install the opam-depext package with 2.1, how opam behave?

Reply:
@psafont I just applied the suggestion!

Reply:
The package is still available, so it will installs it, and can uses it, but its content won't be update (except major issue/security bug). With the new integrated system, there is no more the need to call `opam depext ...`, it's completely transparent. So calling `opam depext` with opam 2.1 will behave as opam 2.0 : install plugin depext and calls it.
The main inconvenience is that you will do an external dependency check twice.

Reply:
> With the new integrated system, there is no more the need to call opam depext ..., it's completely transparent.

Whoa! This is much cooler than I thought!
I'm happy to know that it won't break. I just wanted to make sure that OCaml workflows around the world would not break unless I made changes for opam 2.1 immediately after the release.
Thank you, as always, for all your fantastic work on opam!